server/views/pages/analytics/incentive-levels/prisoners-by-location.njk

```diff
@@ -9,7 +9,11 @@
   <div class="govuk-grid-column-two-thirds">
     {{ govukDetails({
       summaryText: "Help using this chart",
-      text: "Use this chart to see incentive level splits for the prison and wings. What does it tell you about how the policy framework is applied, the population and possible actions for your prison?"
+      text: "Use this chart to see incentive level splits for the prison and wings. What does it tell you about how the policy framework is applied, the population and possible actions for your prison?",
+      attributes: {
+        "data-ga-category": "Help using incentive levels chart",
+        "data-ga-label": user.activeCaseload.id | default("")
```

Comment:
FYI: The label only having the prison ID (without `-` may break the ga dashboard/select logic, shouldn't be the end of the world hopefully tho)